Library/Formula/zeal.rb

```diff
@@ -0,0 +1,43 @@
+class Zeal < Formula
+  desc "Zeal is a simple offline documentation browser inspired by Dash."
+  homepage "http://zealdocs.org/"
+  url "https://github.com/zealdocs/zeal/archive/v0.1.1.tar.gz"
+  sha256 "f4e959f9bc66a6e350ee8a33d34695379633432d103db9776c0c7a76cbc5a9d6"
+  head "https://github.com/zealdocs/zeal.git"
+
+  depends_on "qt5"
+  depends_on "libarchive"
+
+  patch :DATA
+
+  def install
+    qt5 = Formula["qt5"]
+    ENV.append "CFLAGS", "-I#{qt5.installed_prefix}/include -I#{HOMEBREW_PREFIX}/opt/libarchive/include"
```

Comment:
`-I#{Formula["qt5"].opt_include} -I#{Formula["libarchive"].opt_include}` here. The `qt5` one should be being passed automatically. Does it not pick up on that?